db/migrate/20230120170837_create_comment.rb

```diff
@@ -1,9 +1,11 @@
 class CreateComment < ActiveRecord::Migration[7.0]
-  def change
-    create_table :comments do |t|
-      t.string :Text
+   def change
+      create_table :comments do |t|
+         t.string :Text
+         t.datetime :UpdatedAt
+         t.datetime :CreatedAt
 
-      t.timestamps
-    end
-  end
+         t.timestamps
```

Comment:
- [ ] For the correct notation and to avoid future conflicts please 🙏🏽 use snake_case for naming the columns e.g. `text`
- [ ] `t.timestams` will crate updated_at and created_at columns, which means that you don't need to create extra columns, please 🙏🏽  remove UpdateAt and CreateAt columns to avoid future conflicts.